### PR TITLE
Create Word definition page

### PR DESCRIPTION
**Pull request by MySmilingTurban**
This PR adds some new UI elements and code for the page to display the definition of a word.

Screenshot of word definition page

It also changes the layouts (`src/app/layout.tsx` & `src/app/words/layouts.tsx`) that render the underlying pages to keep a consistent coding style with the least amount of redundancy.

### CODE REVIEW

**Comment by amitojsingh** [comment]
@MySmilingTurban could you update the pull request description and include screenshots if possible?

File: src/app/dashboard/page.tsx [hunk 1]
```diff
@@ -4,10 +4,9 @@ import React from 'react'
 
 export default function Dashboard() {  
   return (
-    <main className="flex min-h-screen flex-col justify-between background-layer">
-      <Header loggedIn={true} />
+    <div>
       <div>Dashboard</div>
```
Comment (amitojsingh) [inline src/app/dashboard/page.tsx:8]: Is it possible to create a separate file dedicated to storing all the hardcoded strings used in the project?

File: src/app/profile/page.tsx [hunk 2]
```diff
@@ -3,11 +3,8 @@ import React from 'react'
...
 export default function Login() {
   return (
-    <main className="flex min-h-screen flex-col justify-between background-layer">
-      <Header />
-      <section className="flex flex-row w-full h-full items-center justify-between gap-5 p-12 absolute">
-        Profile
-      </section>
-    </main>
+    <section className="flex flex-row w-full h-full items-center justify-between gap-5 p-12 absolute">
+      Profile
```
Comment (amitojsingh) [inline src/app/profile/page.tsx:7]: Hardcoded string

File: src/app/questions/page.tsx [hunk 3]
```diff
@@ -3,11 +3,8 @@ import React from 'react'
...
 export default function Questions() {
   return (
-    <main className="flex min-h-screen flex-col justify-between background-layer">
-      <Header />
-      <section className="flex flex-row w-full h-full items-center justify-between gap-5 p-12 absolute">
-        Questions
-      </section>
-    </main>
+    <section className="flex flex-row w-full h-full items-center justify-between gap-5 p-12 absolute">
+      Questions
```
Comment (amitojsingh) [inline src/app/questions/page.tsx:7]: hardcoded string

File: src/app/settings/page.tsx [hunk 4]
```diff
@@ -3,11 +3,8 @@ import React from 'react'
...
 export default function Settings() {
   return (
-    <main className="flex min-h-screen flex-col justify-between background-layer">
-      <Header />
-      <section className="flex flex-row w-full h-full items-center justify-between gap-5 p-12 absolute">
-        Settings
-      </section>
-    </main>
+    <section className="flex flex-row w-full h-full items-center justify-between gap-5 p-12 absolute">
+      Settings
```
Comment (amitojsingh) [inline src/app/settings/page.tsx:7]: hardcoded string

**Comment by MySmilingTurban** [comment]
> @MySmilingTurban Veer Ji Could you update the pull request description and include screenshots if possible?

@amitojsingh Added description

**Review by amitojsingh — APPROVED** [review]

**Comment by amitojsingh** [comment]
Good Job veer ji.

**Pull request merged**